jobs/service-fabrik-scheduler/templates/config/settings.yml.erb

```diff
@@ -70,6 +70,15 @@ production:
     protocol: 'http'
     <% end %>
 
+  #############################
+  # DEPLOYMENT HOOKS SETTINGS #
+  #############################
+  deployment_hooks:
```

Comment:
why does scheduler need this config? Hooks IMO, should be sth that only broker must access its an extension of broker but just to sandbox this is being done. So no other external entities must ever communicate with hooks server.

Reply:
Same as above